assignment.hs

```diff
@@ -45,6 +47,8 @@ reciprocal x = . . .
 
 -- your code
 
+oneOverXSquared x = reciprocal(square x)
+
```

Comment:
We normally leave a space after the first function name, just to remind us that the parentheses are not syntax for a function's parameters.

We haven't seen this in class, but you can also write

~~~haskell
oveOverXSquared = reciprocal . square
~~~

That creates a new function that is the composition of the originals.